test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -910,6 +910,71 @@ TEST_F(HttpFilterTest, FilterEnabled) {
             filter_->decodeHeaders(request_headers_, false));
 }
 
+
+// Test that filter can deny for protected path when filter is diabled via filter_enabled field.
```

Comment:
Can you add the integration test I pointed you at as well? Thanks.

Reply:
Added